src/content/en/updates/2017/01/scrolling-intervention.md

```diff
@@ -0,0 +1,150 @@
+in the touchstart or first touchmove events then you will prevent scrolling. The 
+problem is that most often listeners will not call preventDefault(), but the 
+browser needs to wait for the event to finish to be sure of that. Developer 
+defined "passive event listeners" solve this. When you add a touch event with a 
+`{passive: true}` object as the third parameter in your event handler then you are 
+telling the browser that the "touchstart" listener will not call 
+preventDefault() and the browser can safely perform the scroll without blocking 
+on the listener. For example:
+
+    window.addEventListener("touchstart", func, {passive: true} );
+
+## The Intervention
+
+Our main motivation is to reduce the time it takes to update the display after 
+the user touches the screen. To understand the usage of touchstart and touchmove 
+we added metrics to determine how frequently scroll blocking behaviour occurred. 
+
+We looked at the percentage of cancelable touch events that were sent to a root 
+target (window, document, or body) and determined that about 80% of these 
+listeners are conceptually passive but were not registered as such. Given the 
+scale of this problem we noticed a great opportunity to improve scrolling 
+without any developer action by making these events automatically "passive".
+
+This drove us to define our intervention as: if the target of a touchstart or 
+touchmove listener is the `window`, `document` or `body` we default 
+`passive` to `true`. This means that code like: 
+
+    window.addEventListener("touchstart", func);
+
+becomes equivalent to:
+
+    window.addEventListener("touchstart", func, {passive: true} );
+
+Now calls to `preventDefault()` inside the listener will be ignored.
+
+The graph below shows the time taken by the top 1% of scrolls from the time a 
+user touches the screen to scroll to the time the display is updated. This data 
+is for all websites in Chrome for Android. Before the intervention was enabled 
+1% of scrolls took just over 400ms. That has now been reduced to just over 250ms 
+in Chrome 56 Beta; a reduction of about 38%. In the future we hope to make 
+passive true the default for _all_ `touchstart` and `touchmove` listeners, reducing 
+this to below 50ms.
+
+<img src="/web/updates/images/2017/01/scrolling-intervention/image00.png" 
+  class="attempt-float"/>
+
+## Breakage and Guidance
+
+In the vast majority of cases, no breakage will be observed. But when breakage 
+does occur, the most common symptom is that scrolling happens when you don't 
+want it. In rare cases, developers may also notice unexpected **click** events 
+(when `preventDefault()` was missing from a `touchend` listener).
+
+In Chrome 56 and later, DevTools will log a warning when you call 
+`preventDefault()` in an event where the intervention is active.
+
+`touch-passive.html:19 Unable to preventDefault inside passive event listener due 
```

Comment:
I think backticks would work here, but you might do this indented 4 spaces instead